services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js

```diff
@@ -1107,6 +1107,7 @@ qx.Class.define("osparc.dashboard.StudyBrowser", {
 
       let operationPromise = null;
       if (collabGids.length > 1 && amICollaborator) {
+        // Fixme - Study collaborators should be users.
```

Comment:
IN the backend we avoid leaving FIXME and TODO comments. SEE [docs/coding-convensions.md -> CC1](https://github.com/ITISFoundation/osparc-simcore/blob/68a425c36f565acfdc22c1f00000b8183041e2e0/docs/coding-conventions.md#cc1-can-i-use-todo-fixme) Our PR review bot from sonar detects them (i forget most of the time to go through them). 
I would advice to follow the same policy but I most importantly is to agree with @ignapas about this policy.

Reply:
I'll amend it :)